lib/new_relic/agent/custom_event_aggregator.rb

```diff
@@ -27,7 +27,7 @@ def record(type, attributes)
         return unless enabled?
 
         type = @type_strings[type]
-        unless type =~ EVENT_TYPE_REGEX
+        unless EVENT_TYPE_REGEX.match?(type)
```

Comment:
Is it supposed to be `regex.match?(string)`? It seems like in the file below this it's doing `string.match?(regex)`. Although maybe they both do the exact same thing? I'm not sure, just something I noticed

```
EVENT_TYPE_REGEX.match?(type)
# VS
sql&.match?(COMPONENTS_REGEX_MAP[:single_quotes])
```

Reply:
Great question! Both are valid. The `String`, `Regex`, and `Symbol` classes have a `match?` method. The caller/argument change based on the class `match?` is called on.